updated v2.0.3 migrating keys to include a section on switching clients (#398)
prysmaticlabs/documentation: website/versioned_docs/version-2.0.3/advanced/migrating-keys.md

:::danger Good Practice
In order to minimise slashing risk, it is recommended that the migrated validator is not started until at least 1 epoch following the last action taken by the original validator. Epoch status can be monitored on multipe websites, however, typically a new epoch is created approximatley every 6.5 minutes, waiting 7.5 minutes after stopping the original validator should significantly reduce the risk of slashing.  
:::

> ## Switching Clients

Reviewer: ```suggestion
## Guidelines on Switching Between Ethereum Client Software
```

> This portion of the document will provide guidance on switching clients. These steps will be similar to the above steps in Migrating Computers with some key differences. A portion of this guide will be redundant to areas covered elsewhere in the documentation and will link to those places where it is relevant. 

Reviewer: ```suggestion
This portion of the document will provide guidance on switching between Prysm and other consensus clients, such as Teku, Lighthouse, or Nimbus. These steps will be similar to the above steps in Migrating Computers with some key differences. A portion of this guide will be redundant to areas covered elsewhere in the documentation and will link to those places where it is relevant. 
```

> Please see Pryms's note on the importance of client diversity [here.](https://medium.com/prysmatic-labs/prysmatic-labs-statement-on-client-diversity-c0e3c2f05671) 

Reviewer: ```suggestion
Regarding why one may wish to switch to a different client, see our team's note on the importance of client diversity [here](https://medium.com/prysmatic-labs/prysmatic-labs-statement-on-client-diversity-c0e3c2f05671).
```
